tests/lang-ruby.yaml

```
@@ -5,6 +5,7 @@
   - status == 0
   - stdout.indexOf("ruby") != -1
   - stdout.indexOf("2.7.6") != -1 ||
+  - stdout.indexOf("3.0.4") != -1 ||
```

```suggestion
    stdout.indexOf("3.0.4") != -1 ||
```
I think that is why it fails. It adds it as a new command, instead of being part of previous line.
@tnir

---

Oh, that is true... 😭